src/builders/go/gomod2nix/default.nix

```diff
@@ -4,13 +4,13 @@
   externals,
   ...
 }:
-
 {
   fetchedSources,
   dreamLock,
 }:
 let
-  gomod2nixTOML = fetchedSources.mapAttrs
+  gomod2nixTOML =
+    fetchedSources.mapAttrs
     dependencyObject.goName;
```

Comment:
For what reason is that function call pushed into the next line why others are not?

Reply:
It's because there was previously a line between the function and the argument:

```nix
  gomod2nixTOML = fetchedSources.mapAttrs
    dependencyObject.goName;
```

Alejandra interprets that you don't want a long line, and formats it vertically

If you do this:


```nix
  gomod2nixTOML = fetchedSources.mapAttrs dependencyObject.goName;
```

Alejandra will interpret that you want a long line, and would format horizontally

There is no concept of max-line-length, so the user has the freedom of introducing newlines where a break would be more readable